documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1943,6 +1981,19 @@ Each delay is double that of the prior delay, up to the maximum delay. Given the
 |16 |120|20:07
 |===
 
+[id="connector-error-invalid-resume-token"]
+=== Connector Unable to Start - InvalidResumeToken or ChangeStreamHistoryLost
+
+If a connector was stopped for long period and now is unable to start, with the following exception.
+----
+Command failed with error 286 (ChangeStreamHistoryLost): 'PlanExecutor error during aggregation :: caused by :: Resume of change stream was not possible, as the resume point may no longer be in the oplog
+----
+This indicates that the entry which corresponds to the connector's resume token no longer exists in the Oplog. At this point the connector cannot resume streaming, because the oplog no longer contains the previous offset that the connector has processed.
```

Comment:
```suggestion
The preceding exception indicates that the entry that corresponds to the connector's resume token is no longer present in the oplog. 
Because the oplog no longer contains the last offset that the connector processed, the connector cannot resume streaming.
```